### PR TITLE
search & analyze tweets

### PR DESCRIPTION
**Pull request by yongfei25**

### CODE REVIEW

**Comment by yongfei25** [comment]
The sentiment library couldn't recognize most of the words. Results are more lean to neutral.. hmmm..

File: event.json [hunk 1]
```diff
@@ -1,5 +1,4 @@
 {
-  "key3": "value3",
-  "key2": "value2",
-  "key1": "value1"
+  "targetPlaceID": "23424901",
```
Comment (khayuenkam) [inline event.json:2]: what is the difference between here and `handler.js`'s `defaultPlaceId`?

Comment (khayuenkam) [reply event.json:2]: Okie, i get it. Ignore this

File: handler.js [hunk 2]
```diff
@@ -36,20 +42,79 @@ module.exports.hello = (event, context, callback) => {
...
+              console.log('Analyzed text: ', JSON.stringify(item))
+              return item
+            })
+
+          return callback(null, arr)
+        })
+      }, (err, result) => {
+        if (err) return callback(err)
+
+        console.timeEnd('Search & analyze tweets')
```
Comment (khayuenkam) [inline handler.js:85]: Do we need `timeEnd`?

File: handler.js [hunk 3]
```diff
@@ -36,20 +42,79 @@ module.exports.hello = (event, context, callback) => {
...
+          const arr = tweets
+            .filter((t) => {
+              // only return the tweets happened between current and the previous lambda call
+              const tweetTime = (new Date(t.created_at)).getTime()
+              return currentTime - tweetTime < (periodSeconds * 1000)
+            })
+            .map((t) => {
+              // the things we want
+              const goodText = sanitizeText(t.text)
+              const s = sentiment(goodText)
```
Comment (khayuenkam) [inline handler.js:66]: This is trivial. How about this?

``` javascript
const {score, comparative} = sentiment(goodText)
const item = {
   topic: trend.name,
   text: goodText,
   sentiment: {
     score,
     comparative
   }
}
```

**Review by khayuenkam — APPROVED** [review]
LGTM

**Comment by khayuenkam** [comment]
Why is it lean to neutral is it because of different language eg Malay?

File: handler.js (hunk 2 above)
Comment (yongfei25) [reply handler.js:85]: Yes. Calling `timeEnd` will log the message.

**Comment by yongfei25** [comment]
Yeah, Malay/Chinese/Korean, and acronym

File: handler.js (hunk 3 above)
Comment (yongfei25) [reply handler.js:66]: @khayuenkam does this work in node4.3?

Comment (khayuenkam) [reply handler.js:66]: I guess not. I forgot this is node 4.3 :(

**Pull request merged**